### PR TITLE
Added definitions for some techniques

### PR DESCRIPTION
**Pull request by MaaikevS**

### CODE REVIEW

**Review by lzehl — CHANGES_REQUESTED** [review]
@MaaikevS thanks a lot for these definitions. Could you maybe shorten them (definitions should be as short as possible) and move your longer explanations to the description?

File: instances/technique/DABStaining.jsonld [hunk 1]
```diff
@@ -4,11 +4,11 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/DABStaining",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
-  "description": null,
+  "definition": "DAB staining is a technique in which DAB (3, 3'-diaminobenzidine) is oxidized by hydrogen peroxide in a reaction typically catalyzed by horseradish peroxidase (HRP).",
```
Comment (lzehl) [inline instances/technique/DABStaining.jsonld]: I suggest the following for definition:
"In a 'DAB staining', the organic compound DAB (3, 3'-diaminobenzidine) is oxidized in presence of peroxidase and hydrogen peroxide resulting in deposition of a brown, alcohol-insoluble precipitate which can be used in immunohistochemical and blotting applications."

Description can then set to null for now.

Synonym could be "3,3′-Diaminobenzidine staining"

File: instances/technique/NisslStaining.jsonld [hunk 2]
```diff
@@ -4,11 +4,11 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/NisslStaining",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
-  "description": null,
+  "definition": "The Nissl staining is a classic nucleic acid staining technique traditionally used on nervous tissue sections.",
```
Comment (lzehl) [inline instances/technique/NisslStaining.jsonld]: I suggest the following definition:
"The term 'Nissl staining' refers to various basic dyes (e.g., aniline, thionine, cresyl violet) that selectively label negatively charged molecules (e.g., DNA, RNA) and are therefore typically used to highlight important structural features of cell bodies."

I would set description to null for now. 

As synonym I would probably add "Nissl", and "Nissl method".

File: instances/technique/anterogradeTracing.jsonld [hunk 3]
```diff
@@ -4,11 +4,11 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/anterogradeTracing",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
-  "description": null,
+  "definition": "Anterograde tracing is a technique used to trace axonal projections from their source (the cell body or soma) to their point of termination (the synapse).",
+  "description": "Anterograde tracing is a technique used to trace axonal projections from their source (the cell body or soma) to their point of termination (the synapse). Anterograde tracers are taken up by neuronal cell bodies at the injection site and travel to the axon terminals. Anterograde tracing techniques allow for a detailed assessment of neuronal connections between a target population of neurons and their outputs throughout the nervous system.",
```
Comment (lzehl) [inline instances/technique/anterogradeTracing.jsonld]: I would typically not reuse the definition again in the description to avoid redundancy for the reader. Maybe reformulate or delete and switching the order of the remaining description sentences?

File: instances/technique/biocytinStaining.jsonld [hunk 4]
```diff
@@ -4,11 +4,11 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/biocytinStaining",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
-  "description": null,
+  "definition": "Biocytin staining is a technique commonly used in electrophysiology for post-hoc recovery of morphological details of nerve cells.",
+  "description":  "Biocytin staining is a technique commonly used in electrophysiology for post-hoc recovery of morphological details. Biocytin is a histological stain for nerve cells that is included in the recording electrode. It allows for the visualisation of the dendritic arborization and the regions targeted by the axons of the recorded neurons.",
```
Comment (lzehl) [inline instances/technique/biocytinStaining.jsonld]: I would typically not reuse the definition again in the description to avoid redundancy for the reader. Maybe replace definition with:
"In 'biocytin staining' the chemical compound biocytin is used to highlight morphological details of nerve cells." 

I would rephrase the description to:
"Biocytin staining is a technique commonly used in combination with intracellular electrophysiology for post-hoc recovery of morphological details of the recorded neurons. For this, the chemical compound biocytin is included in the recording electrode in order to fill the recorded cell. It allows for the visualisation of the dendritic arborization and the regions targeted by the axons of the recorded neurons."

I would add the synoyms "biocytin filling" and "biocytin labeling"

File: instances/technique/cellAttachedPatchClamp.jsonld [hunk 5]
```diff
@@ -4,11 +4,11 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/cellAttachedPatchClamp",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
-  "description": null,
+  "definition": "Cell-attached patch clamp is a patch clamp recording technique used in electrophysiology in which the intracellular integrity of the cell is preserved.",
```
Comment (lzehl) [inline instances/technique/cellAttachedPatchClamp.jsonld]: I would typically not reuse the definition again in the description to avoid redundancy for the reader. 

I also think the definition is not fully correct or at least misleading. Cell-attached means not only that the intracellular integrity is preserved but also the cell membrane remains intact.

Maybe rephrase definition to: 
"'Cell-attached patch clamp' is an intracellular electrophysiology technique that fully preserves the intracellular integrity by forming a megaohm or gigaohm seal, leaving the cell membrane intact."

I would extract the description from this paper (with ref at the end) https://www.sciencedirect.com/science/article/pii/B9780080552323600121

File: instances/technique/electrocorticography.jsonld [hunk 6]
```diff
@@ -4,11 +4,11 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/electrocorticography",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
+  "definition": "Electrocorticography, or intracranial electroencephalography, is a type of electrophysiological monitoring that uses electrodes placed directly on the exposed surface of the brain to record electrical activity from the cerebral cortex.",
```
Comment (lzehl) [inline instances/technique/electrocorticography.jsonld]: this definition is not fully correct. I would rephrase to:
"'Electrocorticography', short ECoG, is an intracranial electroencephalography technique in which electrodes are placed (subdural or epidural) on the exposed surface of the brain to record electrical activity from the cerebral cortex."

File: instances/technique/extracellularElectrophysiology.jsonld [hunk 7]
```diff
@@ -4,11 +4,11 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/extracellularElectrophysiology",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
+  "definition": "Extracellular recording is an electrophysiology technique that uses an electrode inserted into living tissue to measure electrical activity coming from adjacent cells, usually neurons.",
```
Comment (lzehl) [inline instances/technique/extracellularElectrophysiology.jsonld]: I would rephrase to:
"In 'extracellular electrophysiology' electrodes are inserted into living tissue, but remain outside the cells in the extracellular environment to measure or stimulate electrical activity coming from adjacent cells, usually neurons."

File: instances/technique/electrocorticography.jsonld (hunk 6 above)
Comment (lzehl) [reply instances/technique/electrocorticography.jsonld]: add ECoG to synonyms

File: instances/technique/immunohistochemistry.jsonld [hunk 8]
```diff
@@ -4,11 +4,11 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/immunohistochemistry",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
-  "description": null,
+  "definition": "Immunohistochemistry (IHC) is a method for detecting antigens or haptens in cells of a tissue sections by exploiting the principle of antibodies binding specifically to antigens in biological tissues.",
```
Comment (lzehl) [inline instances/technique/immunohistochemistry.jsonld]: add IHC to synonyms.

I would delete the description and rephrase the defintion to:
"In 'immunohistochemistry' antigens or haptens are detected and visualized in cells of a tissue sections by exploiting the principle of antibodies binding specifically to antigens in biological tissues."

File: instances/technique/optogeneticStimulation.jsonld [hunk 9]
```diff
@@ -4,8 +4,8 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/optogeneticStimulation",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": "Optogenetic stimulation is a genetic technique in which the activity of specific neuron populations is increased using light of a particular wavelength. This is achieved by expressing light-sensitive ion channels, pumps or enzymes specifically in the target cells. ",
-  "description": null,
+  "definition": "Optogenetic stimulation is a genetic technique in which the activity of specific neuron populations is increased using light of a particular wavelength.",
```
Comment (lzehl) [inline instances/technique/optogeneticStimulation.jsonld]: I would delete the description and rephrase the definition to:
"Using light of a particular wavelength, 'optogenetic stimulation' increases or inhibits the activity of neuron populations that express (typically due to genetic manipulation) light-sensitive ion channels, pumps or enzymes."

File: instances/technique/retrogradeTracing.jsonld [hunk 10]
```diff
@@ -4,8 +4,8 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/retrogradeTracing",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
-  "description": null,
+  "definition": "Retrograde tracing is a technique used to trace neural connections from their point of termination (the synapse) to their source (the cell body).",
```
Comment (lzehl) [inline instances/technique/retrogradeTracing.jsonld:7]: see comments for anterograde tracing

**Review by lzehl — CHANGES_REQUESTED** [review]

**Comment by MaaikevS** [comment]
@lzehl I may have misunderstood, but I thought you said to repeat the definition in the description if it was more than 1 sentence, which is why I used the same phrasing. 
As I said before, I wanted to push the population of the definitions/descriptions of the controlled terms, but I am not attached to the exact phrasing. I ticked the box "allow edits by reviewers", so feel free to edit the instances where you see fit.

**Comment by lzehl** [comment]
@MaaikevS I probably did not phrase this part correctly in our meeting: If the first sentence of a very long description is used as well as definition, it does not matter so much. If a description is only 2 / 3 sentence long and one of those matches the definition it seems a bit redundant. Generally, I think it is better to avoid repetition and see the definition as a summary sentence of the whole description. 

I would prefer if you make the changes (feel free to also further adopt), but let me know if you cannot find the time at the moment and want me to take over.

File: instances/technique/cellAttachedPatchClamp.jsonld (hunk 5 above)
Comment (MaaikevS) [reply instances/technique/cellAttachedPatchClamp.jsonld]: Anything in particular? I am not sure what you are looking for

File: instances/technique/biocytinStaining.jsonld [hunk 11]
```diff
@@ -4,11 +4,14 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/biocytinStaining",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
-  "description": null,
+  "definition": "In 'biocytin staining' the chemical compound biocytin is used to highlight morphological details of nerve cells.",
+  "description":  "Biocytin staining is a technique commonly used in combination with intracellular electrophysiology for post-hoc recovery of morphological details of the studied neurons. For this, the chemical compound biocytin is included in the electrode in order to fill the studied cell. It allows for the visualisation of the dendritic arborization and the regions targeted by the axons of the studied neurons.",
```
Comment (lzehl) [inline instances/technique/biocytinStaining.jsonld:8]: I've changed "recorded" to "studied". As far as I know also stimulated neurons can be filled.

**Review by lzehl — APPROVED** [review]
this is a great update thanks (I only made some small changes directly into your fork)
